ReChord_frontend.py

```diff
@@ -40,34 +40,37 @@ def my_form_post():
     Return: rendered result page 'ReChord_result.html' by call on helper functions
     """
 
```

Comment:
I'm okay with you ignoring the pylint warning:

`# pylint: disable=(too-many-return-statements`